src/Plugin/rest/resource/WebformAllFormSubmissions.php

```diff
@@ -112,32 +110,17 @@ public function get(string $webform_id): ModifiedResourceResponse {
 
     $result = ['webform_id' => $webform_id];
 
-    try {
-      $submissionEntityStorage = $this->entityTypeManager->getStorage('webform_submission');
-    }
-    catch (InvalidPluginDefinitionException | PluginNotFoundException $e) {
-      $errors = [
-        'error' => [
-          'message' => $this->t('Could not load webform submission storage'),
-        ],
-      ];
-
-      return new ModifiedResourceResponse($errors, Response::HTTP_INTERNAL_SERVER_ERROR);
-    }
-
-    // Query for webform submissions with this webform_id.
-    $submissionQuery = $submissionEntityStorage->getQuery()
-      ->condition('webform_id', $webform_id);
-
     $requestQuery = $this->currentRequest->query;
 
+    $query = 'SELECT sid, uuid FROM webform_submission WHERE webform_id = :webform_id';
```

Comment:
Maybe a comment about why we use SQL and not the entity manager here. Perfomance...